### DISCUSSION TITLE
Unable to Install, Says that A current version is installed, but I'm unable to find it

### DISCUSSION THREAD

**Discussion by ExultedOne** [SafeExamBrowser/seb-win-refactoring#804] [Q&A]
Hello, I'm trying to install SEB for a class to use on an exam later this week, but whenever I try to install, I'm getting error 0x80070666 saying that I have another version of the software installed, but after extensive searching, I'm not sure that I do. I don't recall ever using the software before either. I have attached a copy of the log file, as well as a screenshot of the error
![image](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/61814355/9a303609-bd3f-48bd-948f-bb662d31fa3d)
[LogFileSEB.txt](https://github.com/SafeExamBrowser/seb-win-refactoring/files/14281815/LogFileSEB.txt)

**Comment by dbuechel** [comment]
The error refers to the Microsoft Visual C++ Redistributable. Please do make sure that you have the correct one installed (you might want to uninstall any existing versions and install the one linked in our release notes, see https://github.com/SafeExamBrowser/seb-win-refactoring/releases/tag/v3.6.0 > Requirements).

In case that would still fail, try to manually install all the listed dependencies and the use the respective setup (.msi) for your platform instead of the setup bundle (.exe) to install SEB.
